### DISCUSSION TITLE
IPC_523128M8MP.1387_22100622.RLC-820A.IMX415.8MP.REOLINK

### DISCUSSION THREAD

**Discussion by jkldgoefgkljefogeg** [AT0myks/reolink-fw-archive#28] [Firmware sharing]
provided by Reolink support to fix FTPS support

version v3.1.0.1387_22100622

on 3.1.0.956_22041501, FTPS (explicit TLS) does not support encryption on data connection (sending `PROT C`). On 3.1.0.1387_22100622, it has proper FTPS support (sending `PROT P`).

[official link](https://support-d.reolink.com/attachments/token/9fhG99webOHErfL1GES7y7ZFU/?name=IPC_523128M8MP.1387_22100622.RLC-820A.IMX415.8MP.REOLINK.pak)

**Comment by jkldgoefgkljefogeg** [comment]
FTPS is broken on this firmware (creating 0 sized files, clear stream not loading), do not use it

**Comment by AT0myks** [comment]
Thanks, added in fb4efb1942c83b5882670c06109a3cc435244284.

**Comment by jkldgoefgkljefogeg** [comment]
working FTPS support in v3.1.0.2685_23091527

**Reply by Nealtron** [reply]
Where can you find v3.1.0.2685_23091527?

**Reply by jkldgoefgkljefogeg** [reply]
I would recommend IPC_523128M8MP.3267_2403051470.RLC-820A.8MP.REOLINK.pak , you can get it from Reolink support
